test/Makefile.am

```diff
@@ -0,0 +1,84 @@
+AM_COLOR_TESTS=always
+AM_CFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir)/inc_nx
+LDADD = ../lib/libnxz.la $(PTHREAD_LIBS) -ldl
+
+test_scripts = test_abi
+
+test_exe = test_adler32 \
+	   test_buf_error \
+	   test_crc32 \
+	   test_inflatesyncpoint \
+	   test_multithread_stress \
+	   test_pid_reuse
+
+selector_tests = test_stress test_deflate test_inflate
```

Comment:
Running these three tests in all modes is taking longer to run make check, although is not that much time I think we could leave just test_stress running on all modes, it tests both `inflate` and `deflate`.

Reply:
Notice that tests can be executed in parallel now. In my tests, it is taking 1/5 of the original time.
I'd also like to split `test_inflate` and `test_deflate` in more tests so that it increases parallelization.
What do you think?